### PR TITLE
[hotfix] fix valueChanges for ie11

### PR DESCRIPTION
**Pull request by yuriykuzin**
On IE `input` event is not triggered on `contenteditable div` and `valueChanges` is not fired. In this fix I use `MutationObserver` to listen text changes.

### CODE REVIEW

**Comment by bampakoa** [comment]
@yuriykuzin Why do we need to uncomment pollyfills?

**Comment by yuriykuzin** [comment]
> @yuriykuzin Why do we need to uncomment pollyfills?

Otherwise demo app doesn't start on IE11

**Comment by bampakoa** [comment]
@yuriykuzin is it better to let the consumer of the library uncomment this for IE?

File: projects/ngx-wig/src/lib/ngx-wig.component.ts [hunk 1]
```diff
@@ -107,6 +110,28 @@ export class NgxWigComponent implements OnInit, OnChanges, ControlValueAccessor
     }
   }
 
+  public ngAfterViewInit(): void {
+    // Workaround for IE11 which doesn't fire 'input' event on
+    // contenteditable
+    // https://stackoverflow.com/a/49287032/7369511
+
+    const isIE = window.document['documentMode'];
```
[the remark this answers is not in the document]
Comment (yuriykuzin) [reply projects/ngx-wig/src/lib/ngx-wig.component.ts]: 👍

**Comment by yuriykuzin** [comment]
@bampakoa good point, fixed. Thanks!

**Pull request merged**